src/components/my-history/custom-my-history.vue

```diff
@@ -22,6 +22,16 @@
 
             <div class="x-flex x-flex-col x-gap-2">
               <p>{{ suggestion.query }}</p>
+              <ul v-if="suggestion.selectedFilters" class="x-flex x-gap-8">
+                <li
+                  v-for="filter in suggestion.selectedFilters"
+                  :key="filter.id"
+                  class="x-text1-sm x-text-auxiliary-50"
```

Comment:
`auxiliary-50` on white background doesn't pass the WCAG contrast test. Maybe better to have it the same as the timestamp.